e2e/atlas/data_lake_test.go

```diff
@@ -30,7 +30,8 @@ import (
 
 func TestDataLakes(t *testing.T) {
 	cliPath, err := e2e.Bin()
-	require.NoError(t, err)
+	r := require.New(t)
+	r.NoError(err)
```

Comment:
[q] Why this is better than `equire.NoError(t, err)`?

Reply:
it's a recommendation the lib gives to people  when using `assert.Something` or `require.Something` multiple times, I haven't gone too deep into the advantages but one will cache the `t` helper so it may reduce some memory usage